libs/remixd/src/services/slitherClient.ts

```diff
@@ -14,7 +14,7 @@ export class SlitherClient extends PluginClient {
 
   constructor (private readOnly = false) {
     super()
-    this.methods = ['analyse']
+    this.methods = ['analyse', 'install']
```

Comment:
you don't need to add `install` here if you don't want to use it from the UI. I am not even sure it's necessary at all.

Reply:
it is removed